apps/editor/src/markdown/mdEditor.ts

```diff
@@ -96,9 +96,18 @@ export default class MdEditor extends EditorBase {
       const items = clipboardData && clipboardData.items;
 
       if (items) {
-        const imageBlob = pasteImageOnly(items);
+        let containsRtf = false;
+        for (var i = 0; i < items.length; i++) {
+          const item = items[i];
+          if (item.kind === 'string' && item.type === 'text/rtf') {
+            containsRtf = true;
+            break;
+          }
+        }
```

Comment:
I think it would be simpler and readable to change it like below :)
```js
import { Transaction } from 'prosemirror-state';
import { EditorView } from 'prosemirror-view';
import { Fragment, Slice } from 'prosemirror-model';
import { ReplaceAroundStep } from 'prosemirror-transform';
import { MdPos, ToastMark } from '@toast-ui/toastmark';
// add below dependency
import toArray from 'tui-code-snippet/collection/toArray';

....
if (items) {
  const containRtfItem = toArray(items).some(item => item.kind === 'string' && item.type === 'text/rtf');
  ...
}
```

Reply:
ah didn't know you have a toArray helper, changed and pushed